### ISSUE TITLE
The mod description is non-user friendly.

### ISSUE THREAD

**Issue by MbtMarco** [FrozenBlock/WilderWild#436]
The description is too long, a ‘casual’ user will never read it all. It should be shortened and summarised, keeping only the main points.

**Comment by Poopooracoocoo** [comment]
Agreed. It should make use of spoilers. And they could create a wiki

**Comment by Rebel459** [comment]
I also agree. The mod description does not do the quality of the mod itself justice imo, as it's not formatted thus that the average user will be interested after reading it. Def needs a more normal-style summary and then spoilers for the full list

**Comment by AViewFromTheTop** [comment]
I don’t think a single person on earth disagrees lmao
Luth said she would try to work on this soon already
It’s been planned
